### PR TITLE
Mobile hamburger menu

### PR DESCRIPTION
**Pull request by lazyCode-404**
### Things archived in this milestone: 
- Added mobile menu 
- Styled the menu
- Added javascript code to script file
- Added background images to the menu
- Updated linters.yml file
- Updated README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,23 +28,24 @@
...
-      <div class="bars">
-        <i class="fas fa-bars bars"></i>
+      <div id="myNav" class="overlay">
+        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+        <div class="overlay-content">
+          <a href="javascript:void(0);" onclick="menuOptions('#portfolio')">Portfolio<i class="fa fa-angle-right"></i></a>
+          <a href="javascript:void(0);" onclick="menuOptions('#about')">About<i class="fa fa-angle-right"></i></a>
+          <a href="javascript:void(0);" onclick="menuOptions('#contact-us')">Contact<i class="fa fa-angle-right"></i></a>
+        </div>
+      </div>
```
Comment (thecodechaser) [inline index.html]: - [ ] You did a good job on the project functionalities but there is a little issue, When I click on the portfolio I am being redirected to the about section, and when I click on the about I am being redirected to the projects section. Please fix this issue.

File: style.css [hunk 2]
```diff
@@ -401,6 +401,70 @@ body {
...
+.overlay a:focus {
+  color: #f1f1f1;
+}
+
+.overlay .closebtn {
+  position: absolute;
+  top: 20px;
+  right: 45px;
+  font-size: 60px;
+}
```
Comment (thecodechaser) [inline style.css]: - [ ] Please reduce the size and right padding of the close(X) button and make sure it matches with the Figma design.

### Your design

image

### Figma design

image

File: style.css [hunk 3]
```diff
@@ -401,6 +401,70 @@ body {
...
+  background-repeat: no-repeat, no-repeat;
+}
+
+.overlay-content {
+  position: relative;
+  top: 25%;
+  width: 100%;
+  text-align: center;
+  margin-top: 30px;
+}
```
Comment (thecodechaser) [inline style.css]: - [ ] Please fix the below issues and make sure your design matches the Figma design.
 - Please reduce the top margin for the mobile menu.
 - Please reduce the size of mobile menu options
 - Please use the correct color for arrows.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @BwalyaNonde, and @bigmosi👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by JuliCarracedo — APPROVED** [review]
Hi @BwalyaNonde , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
[✔️](https://emojipedia.org/check-mark/) No linter errors
[✔️](https://emojipedia.org/check-mark/) Great implementation of the hamburger menu!
[✔️](https://emojipedia.org/check-mark/) Professional PR and repository

## Optional suggestions

- [ OPTIONAL ] Remember that it is a requirement to have instructions on how to set up your projects locally in your README. Kindly add this information.
- [ OPTIONAL ] In any project where you have a package.json file, you must add the node modules to your .gitignore file to avoid uploading thousands of files to GitHub.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**